static/app/components/gridEditable/styles.tsx

```diff
@@ -95,24 +95,27 @@ export const Grid = styled('table')<{height?: string | number; scrollable?: bool
       : ''}
 `;
 
-export const GridRow = styled('tr')`
-  display: contents;
-
-  &:last-child,
-  &:last-child > td:first-child,
-  &:last-child > td:last-child {
-    border-bottom-left-radius: ${p => p.theme.borderRadius};
-    border-bottom-right-radius: ${p => p.theme.borderRadius};
-  }
-`;
-
 /**
  * GridHead is the collection of elements that builds the header section of the
  * Grid. As the entirety of the add/remove/resize actions are performed on the
  * header, most of the elements behave different for each stage.
  */
 export const GridHead = styled('thead')`
-  display: contents;
+  display: grid;
+  grid-template-columns: subgrid;
+  grid-column: 1/-1;
```

Comment:
This is the magic. This element now takes up the full row, and is a grid itself where it's columns are inherited from the parent (using `subgrid`)

Reply:
That is so good, it means we can have proper hover states for entire rows right?